### DISCUSSION TITLE
Video call not working properly in android and ios

### DISCUSSION THREAD

**Discussion by shahrukh339** [community/community#113417] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Problem face in android 

Video is not work properly in Foreground, background and Kill mode,
Call is not connected in when app is kill state 

problem face in iOS 

Not able to see video in foreground ,background  and kill mode 

not able to connect call in background and when app in kill state 

and after 2 to 3 sec call automatically disconnected 

requirement

Azure calling connect in all state like foreground background and kill mode in both android and iOS 

audio and video work properly in all state foreground background and kill mode in both android and iOS 

video call same looking like watsapp call  Skype call

**Comment by queenofcorgis** [comment]
Hi @shahrukh339 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on your video calling platform. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by shahrukh339** [reply]
Where I put my issue because I face many issue with this POC
